### PR TITLE
Add load_study_set command

### LINKED ISSUE #327: Create management command to load multiple studies
### Feature Inspiration

A simple way to load multiple studies, such as the growing collection in https://github.com/PrincetonUniversity/tracebase-rabinowitz-data would be very useful. Being able to control the order studies are loaded is important, since some studies might add compounds used in later studies, etc.

This would also be useful for making it simple to test the loading of various studies in the `tracebase-rabinowitz-data` repo as well as creating a simple and quick way for developers to setup a working database with a variety of edge cases populated in a new database.

### Feature Description

A management command in tracebase (e.g. `load_study_set`) that takes a file of file names that can be used as input to `load_study`. 

### Alternatives Considered

A bash script could work, but it would need to know the location of the `manage.py` script as well as the location of each of the study loading directories.

### Comment

-----

### Assumptions

- List of assumptions made WRT the code
- E.g. We will assume input is correct (explaining why there is no validation)

### Requirements

- List of conditions to be met for the feature
- E.g. Every column/row must display a value, i.e. cannot be empty

### Limitations

- A list of things this work will specifically not do
- E.g. This feature will only handle the most frequent use case X

### Affected/Changed Components

- Files
- Environment variables
- External executables
- Database tables
- Cron job settings
- Etc.

#### GUI Change description

Describe changes the user will see.

#### Code Change Description *(Pseudocode optional)*

Describe code changes planned for the feature.

#### Tests

A test should be planned for each requirement (above), where possible.

### PR DESCRIPTION
**Pull request by lparsons**
## Summary Change Description

Add a command to read a file of file names and use those as
`study_params` for the `load_study` command.

## Affected Issue Numbers

- Resolves #327

## Code Review Notes

This PR is missing tests, though I did test it using the data in https://github.com/PrincetonUniversity/tracebase-rabinowitz-data. I would include a test when we organize example and test data (#172 and #325).

## Checklist

- [x] All issue requirements satisfied (or no linked issues)
- [x] [Linting passes](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting).
- [x] [Migrations created & committed *(or no model changes)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)
- [ ] [Tests implemented *(or no code changes)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
- [x] [All tests pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)

### CODE REVIEW

File: DataRepo/management/commands/load_study_set.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import os
+
+from django.core.management import BaseCommand, call_command
+
+
+class Command(BaseCommand):
+
+    # Show this when the user types help
+    help = (
+        "Loads a set of study using the load_study command. Input is a file of "
```
Comment (hepcat72) [inline DataRepo/management/commands/load_study_set.py:11]:
```suggestion
        "Loads a set of studies using the load_study command. Input is a file of "
```

**Review by hepcat72 — APPROVED** [review]
fabulous.

**Review by fkang-pu — APPROVED** [review]

**Review by jcmatese — APPROVED** [review]
